gemrb/core/Game.cpp

```diff
@@ -40,8 +40,27 @@
 #include "GUI/GameControl.h"
 #include "System/DataStream.h"
 #include "System/StringBuffer.h"
+#include <iterator>
+#include <vector>
+#include <algorithm>
 
 namespace GemRB {
+	
+	
+struct HealingResource {
+	ieResRef resref;
+	Actor* caster;
+	ieWord amount;
+};
+
+struct Injuree {
```

Comment:
On Monday 02 of February 2015 07:23:39 Brad Allred wrote:
> >  namespace GemRB {
> > 
> > +
> > +
> > +struct HealingResource {
> > +	ieResRef resref;
> > +	Actor* caster;
> > +	ieWord amount;
> > +};
> > +
> > +struct Injuree {
> 
> "Injuree" isn't a word... you mean Injured
I made it up, so it's clear it is not an adjective with ellipsis. I think it's 
a french pattern originally.

LP